etc/adminhtml/system/main.xml

```diff
@@ -28,8 +28,10 @@
         <source_model>Adyen\Payment\Model\Config\Source\DemoMode</source_model>
         <config_path>payment/adyen_abstract/demo_mode</config_path>
     </field>
-    <field id="shopper_locale" translate="label" type="text" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1">
-        <label>Locale</label>
+    <field id="shopper_locale" translate="label" type="select" sortOrder="100" showInDefault="1" showInWebsite="1" showInStore="1">
+        <label>Language locale (optional)</label>
+        <tooltip>This will override the default shopper locale of the Magento store (e.g. Dutch (Netherlands)).</tooltip>
```

Comment:
I think I missed this on the first PR.

This field's label and tooltip are different from the prototype from product design: https://youtrack.is.adyen.com/issue/PW-1819
```suggestion
        <label>Locale</label>
```

Reply:
@AlexandrosMor can you fix this ?